artiq/gateware/eem.py

```diff
@@ -53,13 +53,22 @@ def add_std(cls, target, eem, ttl03_cls, ttl47_cls, iostandard=default_iostandar
             phy = ttl03_cls(pads.p, pads.n, dci=dci)
             phys.append(phy)
             target.submodules += phy
-            target.rtio_channels.append(rtio.Channel.from_phy(phy))
+            if i in custom_ififo_lengths:
```

Comment:
Did you test this?

Reply:
Testing now and I see why it would not work.